src/utils/clmfates_interfaceMod.F90

```diff
@@ -80,6 +80,7 @@ module CLMFatesInterfaceMod
    use clm_varctl        , only : use_lch4
    use clm_varctl        , only : fates_history_dimlevel
    use clm_varctl        , only : nsrest, nsrBranch
+   use clm_varctl        , only : CNAllocate_Carbon_only
```

Comment:
Since this method is in clm_varctl I think I like the use of it for FATES. But, I think the CN prefix makes it confused with CN BGC methods, which is explicitly off for FATES. So I think if we also use this method for FATES -- we should change the name to drop the "CN" prefix.